components/key_collector.rst

```diff
@@ -88,36 +94,44 @@ Automations:
 -------------------------------
 
 This action activates a ``key_collector``.  It will start accepting keys.
-If there is more than one key collector, you will need to provide the ``id`` of the one to enable.
+If there is more than one key collector, you need to provide the ``id`` of the one to enable.
 
 .. code-block:: yaml
 
     on_...:
-      then:
-        - key_collector.enable:
+      - key_collector.enable:
+
+Can also be written in lambdas: ``id(pincode_reader).set_enabled(true);``
 
 ``key_collector.disable`` Action
 --------------------------------
 
 This action deactivates a ``key_collector``.  It will stop accepting keys and will clear any already collected ones.
-If there is more than one key collector, you will need to provide the ``id`` of the one to disable.
+If there is more than one key collector, you need to provide the ``id`` of the one to disable.
 
 .. code-block:: yaml
 
     on_...:
-      then:
-        - key_collector.disable:
```

Comment:
Why remove the indent here? String yaml syntax requires it.

Reply:
The yaml is still valid in most cases, but it's an unnecessary change and could cause problems for people.